### ISSUE TITLE
Direct App Insights Kusto queries

### ISSUE THREAD

**Issue by mrtristan** [grafana/grafana#17337]
**What would you like to be added**:
I'd like to be able to write my own Kusto queries in hitting App Insights.

**Why is this needed**:
I cannot currently mold the data as i need to using the existing query-builder form for App Insights.

Potentially related: https://github.com/grafana/grafana/issues/14942

**Comment by torkelo** [comment]
@daniellee do you know if this is the same request as https://github.com/grafana/grafana/issues/14942  (ie a raw query mode)

**Comment by mrtristan** [comment]
@torkelo  yea i wasn't sure if this overlaps with that. was a little hard to parse that description. 

any idea if this _was_ a feature that then went away? in my research on this subject i found the following:

https://github.com/grafana/grafana/commit/dd8ca70151672293b48267a800b2e3682e8a79c5

which _seems_ to do it if the `ctrl.target.appInsights.rawQuery` is truthy

**Comment by torkelo** [comment]
dupe of  #14942
